snow/engine/snowman/transitive.go

```diff
@@ -24,10 +24,14 @@ import (
 	"github.com/ava-labs/avalanchego/utils/bag"
 	"github.com/ava-labs/avalanchego/utils/logging"
 	"github.com/ava-labs/avalanchego/utils/set"
+	"github.com/ava-labs/avalanchego/utils/units"
 	"github.com/ava-labs/avalanchego/utils/wrappers"
 )
 
-const nonVerifiedCacheSize = 128
+const (
+	nonVerifiedCacheSize = 128 * units.MiB
```

Comment:
@StephenButtolph just to very I understand the issue here:
These hyper blocks are beefy and are quickly verified.
With item based cash they are kept in memory until other 128 blocks arrive, which is (much) more than hyperSDK need to verify them.
A size-based cache would saturate faster and keep them less in memory (still longer than needed to verify them).
Is this correct?

Reply:
correct. This cache specifically probably didn't have a problem... But some of the other caches (with thousands of blocks) were causing issues.